x-pack/plugins/ml/server/models/data_recognizer/modules/auditbeat_process_hosts_ecs/ml/hosts_high_count_process_events_ecs.json

```diff
@@ -24,7 +24,7 @@
     "time_format": "epoch_ms"
   },
   "custom_settings": {
-    "created_by": "ml-module-auditd-hosts",
+    "created_by": "ml-module-auditbeat-process-hosts",
```

Comment:
Changing this makes life hard for the telemetry repo because jobs will exist in the wild that have a value of `ml-module-auditd-hosts`, so then the telemetry repo will have to cope with both.

Reply:
Actually, it looks like `ml-module-auditd-hosts` wasn't in 6.7 - is that correct?  If so, I'd be happy to change this provided this PR gets backported to 7.0 in time for 7.0.0-rc2.  The problem only arises if people will have jobs existing in clusters that get upgraded to 7.1 containing `ml-module-auditd-hosts`.

Reply:
I will go with `ml-module-auditbeat-process-hosts` and `ml-module-auditbeat-process-docker` and backport to 7.0.